tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27,py34,py35,py36,py37,pypy,flake8
+
+[testenv]
+deps = pytest>=3.0
```

Comment:
Here's a reason for `requirements.txt`. In tests you ask for pytest >= 3.0.0 while in setup.py this is >= 4.0.0.

I prefer to have this sort of information only in a single place.

In that regard maybe tox isn't the best option here b/c there aren't a lot of environments supported after all. It is fine by me if you want to use it, I am more fan of Makefile and simple test targets (see tap plugin again). Then all of that is just called by Travis.

Reply:
For me, tox is more convenient :) 

Currently, everything is installed via `setup.py`, so dependencies could be omitted here (for now at least)